.github/workflows/main.yml

```diff
@@ -518,3 +518,8 @@ jobs:
       if: startsWith(github.ref, 'refs/tags/')
       with:
         files: "gh-release/*.tar.gz*"
+    - uses: dtolnay/rust-toolchain@stable
```

Comment:
Rust is already installed earlier within this CI job, so you don't need to do it again.

I'm not actually particularly sure why, it doesn't seem to be used and it's also nightly for some reason. But it shouldn't be installed twice, so one of them has to be removed.

Reply:
I removed the original one because I too can't see where it's used, if something is wrong it should fail at regular CI already I believe.